api/storage_service.cc

```diff
@@ -312,6 +312,15 @@ void set_repair(http_context& ctx, routes& r, sharded<repair_service>& repair) {
                 });
     });
 
+    ctx.db.local().get_builtin_routine_registry().register_routine("keyspace_repair",
+            db::make_single_value_routine<int>("repair", [&ctx, &repair](std::unordered_map<sstring, sstring>&& val) {
```

Comment:
The correct type for cql int is int32_t. It's the same thing, but let's use the same name as the definition.